template/python3-http-armhf/index.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, request, jsonify
```

Comment:
No header?
I think it would be a good practice to put 
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
```
everywhere

Reply:
I'm unfamiliar with this so I'll take a look into it

Reply:
We haven't done this elsewhere so please propose it generically @pcorbel in a separate issue.